docker-compose.yml

```diff
@@ -93,6 +93,17 @@ services:
       - /bin/entrypoint.sh
     command: ["run"]
 
+  cypress-gui:
+    <<: *cypress
+    environment:
+      - DISPLAY
+      - CYPRESS_BASE_URL=http://cypressserver:3000
+    volumes:
+      - .docker/cypress/spec:/e2e
+      - .docker/cypress/entrypoint.sh:/bin/entrypoint.sh
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+    command: ["open"]
```

Comment:
Hier sollten wir noch etwas äquivalentes zu `--project .` anhängen das in Cypress 3 noch funktionierte. Sonst muss man beim Starten des GUI immer manuell das Projekt suchen gehen

Reply:
Tönt gut. Bei mir funktioniert `--project .` genauso…

Reply:
Ja habe es gerade ausprobiert und musste das ergänzen `command: ["open", "--project", "."]`

Reply:
Zudem musste ich bei mir noch dio Option `ipc: host` ergänzen weil mir sonst Chrome gecrasht ist. Wahrscheinlich zu wenig shared memory. Würde ich aber so nicht aufnehmen, höchstens als zusätzlicher Hinweis.

Reply:
hmm, ich wäre sowieso dafür per default electron zu verwenden. macht im container weniger Probleme…